diff -ur OpenCSG-1.5.0.orig/example/Makefile OpenCSG-1.5.0/example/Makefile
--- OpenCSG-1.5.0.orig/example/Makefile	2022-04-10 17:04:35.000000000 +0200
+++ OpenCSG-1.5.0/example/Makefile	2022-04-11 01:16:42.507766104 +0200
@@ -16,7 +16,7 @@
 INCPATH       = -I. -I../include -I/usr/X11R6/include
 LINK          = g++
 LFLAGS        = -Wl,-O1
-LIBS          = $(SUBLIBS)  -L/usr/X11R6/lib64 -L../lib -lopencsg -lGLEW -lGLU -lglut -lGL 
+LIBS          = $(SUBLIBS) -L/app/lib -L/usr/X11R6/lib64 -L../lib -lopencsg -lGLEW -lGLU -lglut -lGL 
 AR            = ar cqs
 RANLIB        = 
 QMAKE         = /usr/lib/x86_64-linux-gnu/qt4/bin/qmake
@@ -124,13 +124,13 @@
 ####### Install
 
 install_target: first FORCE
-	@$(CHK_DIR_EXISTS) $(INSTALL_ROOT)/usr/local/bin/ || $(MKDIR) $(INSTALL_ROOT)/usr/local/bin/ 
-	-$(INSTALL_PROGRAM) "$(QMAKE_TARGET)" "$(INSTALL_ROOT)/usr/local/bin/$(QMAKE_TARGET)"
-	-$(STRIP) "$(INSTALL_ROOT)/usr/local/bin/$(QMAKE_TARGET)"
+	@$(CHK_DIR_EXISTS) $(INSTALL_ROOT)/app/bin/ || $(MKDIR) $(INSTALL_ROOT)/app/bin/ 
+	-$(INSTALL_PROGRAM) "$(QMAKE_TARGET)" "$(INSTALL_ROOT)/app/bin/$(QMAKE_TARGET)"
+	-$(STRIP) "$(INSTALL_ROOT)/app/bin/$(QMAKE_TARGET)"
 
 uninstall_target:  FORCE
-	-$(DEL_FILE) "$(INSTALL_ROOT)/usr/local/bin/$(QMAKE_TARGET)"
-	-$(DEL_DIR) $(INSTALL_ROOT)/usr/local/bin/ 
+	-$(DEL_FILE) "$(INSTALL_ROOT)/app/bin/$(QMAKE_TARGET)"
+	-$(DEL_DIR) $(INSTALL_ROOT)/app/bin/ 
 
 
 install:  install_target  FORCE
diff -ur OpenCSG-1.5.0.orig/src/batch.cpp OpenCSG-1.5.0/src/batch.cpp
--- OpenCSG-1.5.0.orig/src/batch.cpp	2022-04-10 17:04:35.000000000 +0200
+++ OpenCSG-1.5.0/src/batch.cpp	2022-04-11 02:40:14.442703428 +0200
@@ -44,7 +44,7 @@
 
         FullscreenPrimitive fullscreen;
 
-        const size_t numberOfPrimitives = primitives.size();
+        const std::size_t numberOfPrimitives = primitives.size();
         mBatches.reserve(numberOfPrimitives);
 
         std::vector<Batch> batchCandidates;
@@ -118,7 +118,7 @@
         return mBatches.end();
     }
 
-    size_t Batcher::size() const {
+    std::size_t Batcher::size() const {
         return mBatches.size();
     }
 
diff -ur OpenCSG-1.5.0.orig/src/batch.h OpenCSG-1.5.0/src/batch.h
--- OpenCSG-1.5.0.orig/src/batch.h	2022-04-10 17:04:35.000000000 +0200
+++ OpenCSG-1.5.0/src/batch.h	2022-04-11 02:40:42.394989217 +0200
@@ -28,6 +28,7 @@
 
 #include "opencsgConfig.h"
 #include <vector>
+#include <cstddef>
 
 namespace OpenCSG {
 
@@ -46,7 +47,7 @@
         /// returns end of batches
         std::vector<Batch>::const_iterator end() const;
         /// return number of batches
-        size_t size() const;
+        std::size_t size() const;
 
     private:
         std::vector<Batch> mBatches;
diff -ur OpenCSG-1.5.0.orig/src/Makefile OpenCSG-1.5.0/src/Makefile
--- OpenCSG-1.5.0.orig/src/Makefile	2022-04-10 17:04:35.000000000 +0200
+++ OpenCSG-1.5.0/src/Makefile	2022-04-11 01:16:42.507766104 +0200
@@ -16,7 +16,7 @@
 INCPATH       = -I. -I../include -I.. -I/usr/X11R6/include
 LINK          = g++
 LFLAGS        = -Wl,-O1 -shared -Wl,-soname,libopencsg.so.1
-LIBS          = $(SUBLIBS)  -L/usr/X11R6/lib64 -lGLEW -lGL 
+LIBS          = $(SUBLIBS) -L/app/lib -L/usr/X11R6/lib64 -lGLEW -lGL 
 AR            = ar cqs
 RANLIB        = 
 QMAKE         = /usr/lib/x86_64-linux-gnu/qt4/bin/qmake
@@ -285,29 +285,29 @@
 ####### Install
 
 install_target: first FORCE
-	@$(CHK_DIR_EXISTS) $(INSTALL_ROOT)/usr/local/lib/ || $(MKDIR) $(INSTALL_ROOT)/usr/local/lib/ 
-	-$(INSTALL_PROGRAM) "../lib/$(TARGET)" "$(INSTALL_ROOT)/usr/local/lib/$(TARGET)"
-	-$(STRIP) --strip-unneeded "$(INSTALL_ROOT)/usr/local/lib/$(TARGET)"
-	-$(SYMLINK) "$(TARGET)" "$(INSTALL_ROOT)/usr/local/lib/$(TARGET0)"
-	-$(SYMLINK) "$(TARGET)" "$(INSTALL_ROOT)/usr/local/lib/$(TARGET1)"
-	-$(SYMLINK) "$(TARGET)" "$(INSTALL_ROOT)/usr/local/lib/$(TARGET2)"
+	@$(CHK_DIR_EXISTS) $(INSTALL_ROOT)/app/lib/ || $(MKDIR) $(INSTALL_ROOT)/app/lib/ 
+	-$(INSTALL_PROGRAM) "../lib/$(TARGET)" "$(INSTALL_ROOT)/app/lib/$(TARGET)"
+	-$(STRIP) --strip-unneeded "$(INSTALL_ROOT)/app/lib/$(TARGET)"
+	-$(SYMLINK) "$(TARGET)" "$(INSTALL_ROOT)/app/lib/$(TARGET0)"
+	-$(SYMLINK) "$(TARGET)" "$(INSTALL_ROOT)/app/lib/$(TARGET1)"
+	-$(SYMLINK) "$(TARGET)" "$(INSTALL_ROOT)/app/lib/$(TARGET2)"
 
 uninstall_target:  FORCE
-	-$(DEL_FILE) "$(INSTALL_ROOT)/usr/local/lib/$(TARGET)" 
-	 -$(DEL_FILE) "$(INSTALL_ROOT)/usr/local/lib/$(TARGET0)" 
-	 -$(DEL_FILE) "$(INSTALL_ROOT)/usr/local/lib/$(TARGET1)" 
-	 -$(DEL_FILE) "$(INSTALL_ROOT)/usr/local/lib/$(TARGET2)"
-	-$(DEL_DIR) $(INSTALL_ROOT)/usr/local/lib/ 
+	-$(DEL_FILE) "$(INSTALL_ROOT)/app/lib/$(TARGET)" 
+	 -$(DEL_FILE) "$(INSTALL_ROOT)/app/lib/$(TARGET0)" 
+	 -$(DEL_FILE) "$(INSTALL_ROOT)/app/lib/$(TARGET1)" 
+	 -$(DEL_FILE) "$(INSTALL_ROOT)/app/lib/$(TARGET2)"
+	-$(DEL_DIR) $(INSTALL_ROOT)/app/lib/ 
 
 
 install_headers: first FORCE
-	@$(CHK_DIR_EXISTS) $(INSTALL_ROOT)/usr/local/include/ || $(MKDIR) $(INSTALL_ROOT)/usr/local/include/ 
-	-$(INSTALL_FILE) ../include/opencsg.h $(INSTALL_ROOT)/usr/local/include/
+	@$(CHK_DIR_EXISTS) $(INSTALL_ROOT)/app/include/ || $(MKDIR) $(INSTALL_ROOT)/app/include/ 
+	-$(INSTALL_FILE) ../include/opencsg.h $(INSTALL_ROOT)/app/include/
 
 
 uninstall_headers:  FORCE
-	-$(DEL_FILE) -r $(INSTALL_ROOT)/usr/local/include/opencsg.h
-	-$(DEL_DIR) $(INSTALL_ROOT)/usr/local/include/ 
+	-$(DEL_FILE) -r $(INSTALL_ROOT)/app/include/opencsg.h
+	-$(DEL_DIR) $(INSTALL_ROOT)/app/include/ 
 
 
 install:  install_target install_headers  FORCE
diff -ur OpenCSG-1.5.0.orig/src/renderSCS.cpp OpenCSG-1.5.0/src/renderSCS.cpp
--- OpenCSG-1.5.0.orig/src/renderSCS.cpp	2022-04-10 17:04:35.000000000 +0200
+++ OpenCSG-1.5.0/src/renderSCS.cpp	2022-04-11 02:41:10.643277814 +0200
@@ -299,7 +299,7 @@
 
         void renderIntersectedFront(const std::vector<Primitive*>& primitives) {
 
-            const size_t numberOfPrimitives = primitives.size();
+            const std::size_t numberOfPrimitives = primitives.size();
 
             glDepthMask(GL_TRUE);
 
@@ -446,7 +446,7 @@
         bool subtractPrimitivesWithOcclusionQueries(std::vector<Batch>::const_iterator begin,
                                                     std::vector<Batch>::const_iterator end) {
 
-            const size_t numberOfBatches = end - begin;
+            const std::size_t numberOfBatches = end - begin;
             if (numberOfBatches == 0) {
                 return true;
             }
