diff -ru orig/example/Makefile mod/example/Makefile
--- orig/example/Makefile	2023-09-15 20:15:20.000000000 +0200
+++ mod/example/Makefile	2023-10-05 11:15:42.869005811 +0200
@@ -38,7 +38,7 @@
 COMPRESS      = gzip -9f
 LINK          = g++
 LFLAGS        = -Wl,-O1
-LIBS          = $(SUBLIBS) -L../lib -lopencsg -lGLU -lglut /usr/lib/x86_64-linux-gnu/libGL.so   
+LIBS          = $(SUBLIBS) -L/app/lib -L/usr/X11R6/lib64 -L../lib -lopencsg -lGLEW -lGLU -lglut -lGL
 AR            = ar cqs
 RANLIB        = 
 SED           = sed
@@ -256,13 +256,13 @@
 ####### Install
 
 install_target: first FORCE
-	@test -d $(INSTALL_ROOT)/usr/local/bin || mkdir -p $(INSTALL_ROOT)/usr/local/bin
-	-$(QINSTALL_PROGRAM) $(QMAKE_TARGET) $(INSTALL_ROOT)/usr/local/bin/$(QMAKE_TARGET)
-	-$(STRIP) $(INSTALL_ROOT)/usr/local/bin/$(QMAKE_TARGET)
+	@test -d $(INSTALL_ROOT)/app/bin || mkdir -p $(INSTALL_ROOT)/app/bin
+	-$(QINSTALL_PROGRAM) $(QMAKE_TARGET) $(INSTALL_ROOT)/app/bin/$(QMAKE_TARGET)
+	-$(STRIP) $(INSTALL_ROOT)/app/bin/$(QMAKE_TARGET)
 
 uninstall_target: FORCE
-	-$(DEL_FILE) $(INSTALL_ROOT)/usr/local/bin/$(QMAKE_TARGET)
-	-$(DEL_DIR) $(INSTALL_ROOT)/usr/local/bin/ 
+	-$(DEL_FILE) $(INSTALL_ROOT)/app/bin/$(QMAKE_TARGET)
+	-$(DEL_DIR) $(INSTALL_ROOT)/app/bin/ 
 
 
 install: install_target  FORCE
diff -ru orig/src/Makefile mod/src/Makefile
--- orig/src/Makefile	2023-09-15 20:15:20.000000000 +0200
+++ mod/src/Makefile	2023-10-05 11:17:21.836250173 +0200
@@ -17,7 +17,7 @@
 DEFINES       = 
 CFLAGS        = -pipe -O2 -Wall -W -fPIC $(DEFINES)
 CXXFLAGS      = -pipe -O2 -Wall -W -fPIC $(DEFINES)
-INCPATH       = -I. -I../include -I.. -isystem /usr/local/include -I/usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++
+INCPATH       = -I. -I../include -I.. -isystem /app/include -I/usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++
 QMAKE         = /usr/lib/qt5/bin/qmake
 DEL_FILE      = rm -f
 CHK_DIR_EXISTS= test -d
@@ -39,7 +39,7 @@
 DISTNAME      = opencsg1.6.0
 LINK          = g++
 LFLAGS        = -Wl,-O1 -shared -Wl,-soname,libopencsg.so.1
-LIBS          = $(SUBLIBS) -L/usr/local/lib /usr/lib/x86_64-linux-gnu/libGL.so   
+LIBS          = $(SUBLIBS) -L/app/lib -L/usr/X11R6/lib64 -lGLEW -lGL
 AR            = ar cqs
 RANLIB        = 
 SED           = sed
@@ -555,28 +555,28 @@
 ####### Install
 
 install_target: first FORCE
-	@test -d $(INSTALL_ROOT)/usr/local/lib || mkdir -p $(INSTALL_ROOT)/usr/local/lib
-	-$(QINSTALL_PROGRAM) ../lib/$(TARGET) $(INSTALL_ROOT)/usr/local/lib/$(TARGET)
-	-$(STRIP) --strip-unneeded $(INSTALL_ROOT)/usr/local/lib/$(TARGET)
-	-$(SYMLINK) $(TARGET) $(INSTALL_ROOT)/usr/local/lib/$(TARGET0)
-	-$(SYMLINK) $(TARGET) $(INSTALL_ROOT)/usr/local/lib/$(TARGET1)
-	-$(SYMLINK) $(TARGET) $(INSTALL_ROOT)/usr/local/lib/$(TARGET2)
+	@test -d $(INSTALL_ROOT)/app/lib || mkdir -p $(INSTALL_ROOT)/app/lib
+	-$(QINSTALL_PROGRAM) ../lib/$(TARGET) $(INSTALL_ROOT)/app/lib/$(TARGET)
+	-$(STRIP) --strip-unneeded $(INSTALL_ROOT)/app/lib/$(TARGET)
+	-$(SYMLINK) $(TARGET) $(INSTALL_ROOT)/app/lib/$(TARGET0)
+	-$(SYMLINK) $(TARGET) $(INSTALL_ROOT)/app/lib/$(TARGET1)
+	-$(SYMLINK) $(TARGET) $(INSTALL_ROOT)/app/lib/$(TARGET2)
 
 uninstall_target: FORCE
-	-$(DEL_FILE) $(INSTALL_ROOT)/usr/local/lib/$(TARGET) 
-	 -$(DEL_FILE) $(INSTALL_ROOT)/usr/local/lib/$(TARGET0) 
-	 -$(DEL_FILE) $(INSTALL_ROOT)/usr/local/lib/$(TARGET1) 
-	 -$(DEL_FILE) $(INSTALL_ROOT)/usr/local/lib/$(TARGET2)
-	-$(DEL_DIR) $(INSTALL_ROOT)/usr/local/lib/ 
+	-$(DEL_FILE) $(INSTALL_ROOT)/app/lib/$(TARGET) 
+	 -$(DEL_FILE) $(INSTALL_ROOT)/app/lib/$(TARGET0) 
+	 -$(DEL_FILE) $(INSTALL_ROOT)/app/lib/$(TARGET1) 
+	 -$(DEL_FILE) $(INSTALL_ROOT)/app/lib/$(TARGET2)
+	-$(DEL_DIR) $(INSTALL_ROOT)/app/lib/ 
 
 
 install_headers: first FORCE
-	@test -d $(INSTALL_ROOT)/usr/local/include || mkdir -p $(INSTALL_ROOT)/usr/local/include
-	-$(QINSTALL) ../include/opencsg.h $(INSTALL_ROOT)/usr/local/include/opencsg.h
+	@test -d $(INSTALL_ROOT)/app/include || mkdir -p $(INSTALL_ROOT)/app/include
+	-$(QINSTALL) ../include/opencsg.h $(INSTALL_ROOT)/app/include/opencsg.h
 
 uninstall_headers: FORCE
-	-$(DEL_FILE) -r $(INSTALL_ROOT)/usr/local/include/opencsg.h
-	-$(DEL_DIR) $(INSTALL_ROOT)/usr/local/include/ 
+	-$(DEL_FILE) -r $(INSTALL_ROOT)/app/include/opencsg.h
+	-$(DEL_DIR) $(INSTALL_ROOT)/app/include/ 
 
 
 install: install_target install_headers  FORCE
